tests/test_removers_listed_value_remover.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+
+from langworld_db_data.removers.listed_value_remover import (
+    ListedValueRemover,
+    ListedValueRemoverError,
+)
+from tests.helpers import check_existence_of_output_csv_file_and_compare_with_gold_standard
+from tests.paths import (
+    DIR_WITH_REMOVERS_FEATURE_PROFILES,
+    DIR_WITH_REMOVERS_TEST_FILES,
+    INPUT_FILE_WITH_LISTED_VALUES_FOR_REMOVERS,
+    OUTPUT_DIR_FOR_LISTED_VALUE_REMOVER_FEATURE_PROFILES,
+)
+
+GS_FILE_WITH_LISTED_VALUES_REMOVING_THE_LAST = (
+    DIR_WITH_REMOVERS_TEST_FILES / "features_listed_values_gold_standard_for_removing_the_last.csv"
+)
+GS_FILE_WITH_LISTED_VALUES_REMOVING_IN_MIDDLE = (
+    DIR_WITH_REMOVERS_TEST_FILES
+    / "features_listed_values_gold_standard_for_removing_in_middle.csv"
+)
```

Comment:
how about `_the_first`? It could be the same as `_in_middle`, but it's nevertheless expected that we cover both edge cases (first and last)

Reply:
Done!